lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid;
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  max-width: 860px;
+  width: fit-content;
+}
+
+.nx-policy-violation-indicator__badge {
+  border-top-left-radius: $nx-border-radius;
+  border-bottom-left-radius: $nx-border-radius;
+  height: 30px;
+
+  .nx-icon {
+    color: #fff;
+    vertical-align: -5px;
```

Comment:
Rather than this I recommend making `.nx-policy-violation-indicator__badge` a flex container and using `align-items`.

Reply:
> Rather than this I recommend making `.nx-policy-violation-indicator__badge` a flex container and using `align-items`.

https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/cf4b075d9ef14be82fc2383067caec93e3200dba